servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/ManagementClientConstants.java

```diff
@@ -16,10 +16,6 @@
     static String ATOM_NS = "http://www.w3.org/2005/Atom";
     static String SB_NS = "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect";
     static String XML_SCHEMA_INSTANCE_NS = "http://www.w3.org/2001/XMLSchema-instance";
-    static String XML_SCHEMA_NS = "http://www.w3.org/2001/XMLSchema";
```

Comment:
[SpotBugs-P3]

These fields are unused and has package private access level. It is ok to delete them.